lms/djangoapps/discussion_api/api.py

```diff
@@ -127,7 +127,7 @@ def get_thread_list_url(request, course_key, topic_id_list=None, following=False
     path = reverse("thread-list")
     query_list = (
```

Comment:
Just add this to distinguish the change for the migration: ``` Edraak (mobile): encode Arabic or unicode characters in topic_id ``` , or however you see a better message